build.gradle

```diff
@@ -43,6 +45,8 @@ allprojects {
     google()
     maven { url 'https://www.jetbrains.com/intellij-repository/releases' }
     maven { url "https://cache-redirector.jetbrains.com/intellij-dependencies" }
+    maven { url "https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies/" }
+    maven { url "https://maven.pkg.jetbrains.space/public/p/ktor/eap" }
```

Comment:
why are these needed, and can we add a comment so if those dependencies change we can remove these repositories?

Reply:
Unfortunately yes, IntelliJ sometimes used a snapshot version of ktor. The other repo contains the IntelliJ Android plugin (the backported one for ultimate) and newer versions of IntelliJ uses some classes from this plugin. I will add some comments too.